tcms/core/utils/__init__.py

```diff
@@ -17,15 +17,23 @@ def string_to_list(strs, spliter=','):
         str_list = (str(item).strip() for item in strs.split(spliter))
     else:
         str_list = (strs,)
-    return [s for s in str_list if s]
+
+    lst = []
+    for string in str_list:
+        if string:
+            lst.append(string)
+    return lst
 
 
 def form_errors_to_list(form):
     """
     Convert errors of form to list
     Use for Ajax.Request response
     """
-    return [(k, str(v[0])) for k, v in form.errors.items()]
+    errors = []
+    for key, value in form.errors.items():
+        errors.append((key, str(value[0])))
```

Comment:
Isn't `value[x]` already a string ? Why do we have to cast to str.

Reply:
Not sure about that. Will need to investigate further.

Reply:
so what's up with that one ?